File: result/translate/starchat/java2python/python/SELECT_A_RANDOM_NUMBER_FROM_STREAM_WITH_O1_SPACE.py
def f_gold(x):
        res = 0
        count = 0
        count += 1
        if count == 1:
            res = x
        else:
            import random
            i = random.randint(0, count - 1)
            if i == count - 1:
                res = x
        return res

File: dataset/java2python/java/SELECT_A_RANDOM_NUMBER_FROM_STREAM_WITH_O1_SPACE.java
int f_gold ( int x ) {
  int res = 0 ;
  int count = 0 ;
  count ++ ;
  if ( count == 1 ) res = x ;
  else {
    Random r = new Random ( ) ;
    int i = r . nextInt ( count ) ;
    if ( i == count - 1 ) res = x ;
  }
  return res ;
}